hoprd/db/api/Cargo.toml

```diff
@@ -0,0 +1,48 @@
+[package]
```

Comment:
Pls run the `cargo-machete` to identify whether all of these packages are really needed also for the `hoprd/db`

Reply:
The dependencies listed by `machete` are actually used. Do you know an other crate doing a better job at this ?